string imu_port_name
uint8 VI_FIFO_matcher
float64 imu_acc_bias_X
float64 imu_acc_bias_Y
float64 imu_acc_bias_Z